Source/CarthageKit/XCDBLDExtensions.swift

```diff
@@ -1,33 +1,25 @@
-import Foundation
 import ReactiveSwift
-
-/// Describes how to locate the actual project or workspace that Xcode should
-/// build.
-public enum ProjectLocator: Comparable {
```

Comment:
Some parts of these types didn't fit will in XCDBLD, so they now live in extensions here. Some of them can probably move eventually, but it will require some work to do so.